### PR TITLE
Take system message into account for unread message divider

### PR DESCRIPTION
**Pull request by igorkravchenko**
Address wrong placement of new message divider by taking system message into account.

MOB-2151

### CODE REVIEW

**Review by rasmustautsglia — APPROVED** [review]

File: GliaWidgetsTests/ViewModel/Chat/ChatItemTests.swift [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+            operatorMessage,
+            systemMessage,
+            unreadMessageDivider,
+            visitorMessage,
+            transferring
+        ]
+        for kind in kinds {
+            let chatItem = ChatItem(kind: kind)
+            switch kind {
+            case .callUpgrade:
```
Comment (gersonnoboa) [inline GliaWidgetsTests/ViewModel/Chat/ChatItemTests.swift:33]: I think the ones that need to be true can be grouped into one case, no? and then the same for the false ones

**Review by gersonnoboa — APPROVED** [review]

File: GliaWidgetsTests/ViewModel/Chat/ChatItemTests.swift (hunk 1 above)
Comment (igorkravchenko) [reply GliaWidgetsTests/ViewModel/Chat/ChatItemTests.swift]: Indeed a84c9d6236615aba80dddccce0a80b7dd616247f.

**Comment by igorkravchenko** [comment]
!squash

**Comment by igorkravchenko** [comment]
!merge

**Pull request merged**